src/components/QuizGameApi/index.jsx

```diff
@@ -0,0 +1,18 @@
+// we fetch those questions from opentdb website. we can pick the type of questions,the amount and the dificulty.
+// here we have a 10 questions as an amount, nature as a category ,easy, middle and hard for the difficulty and multiple choices as a type.
+// iam fetching data from a website called opentdb.com this website will give us a simple questions about the nature.
+// this is an example of the game https://lovattspuzzles.com/online-puzzles-competitions/ultimate-online-trivia-quiz.
+import shuffle from '../../components/memorygame/shuffle.jsx'
+
+export const fetchQuizQuestions = async (amount, difficulty) => {
+  const natureCategory = 17
```

Comment:
const NATURE_CATEGORY  and move below the imports

Reply:
also run yarn run fix

Reply:
PTAL